### PR TITLE
Failing test for NH-3681

### PR DESCRIPTION
**Pull request by DanielLaberge**
Added a simplified failing test case for https://nhibernate.jira.com/browse/NH-3681

### CODE REVIEW

File: src/NHibernate.Test/Linq/ByMethod/GroupByTests.cs [hunk 1]
```diff
@@ -507,6 +507,19 @@ public void ProjectingWithSubQueriesFilteredByTheAggregateKey()
 			Assert.That(result[15].FirstOrder, Is.EqualTo(10255));
 		}
 
+		[Test(Description = "NH-3681"), Ignore("NH-3681 not yet fixed")]
+		public void SelectManyGroupByAggregateProjection()
```
Comment (hazzik) [inline src/NHibernate.Test/Linq/ByMethod/GroupByTests.cs]: Please replace [Ignore] with [KnownBug]

**Comment by hazzik** [comment]
@DanielLaberge please rebase (and squash) your changes on top of master.

**Comment by DanielLaberge** [comment]
Here you go @hazzik

**Comment by DanielLaberge** [comment]
@hazzik , do you need anything else from me? Also, why does github show 4 test configurations failing?

**Comment by hazzik** [comment]
@DanielLaberge they are failing because of different thing. Probably in future you'll need to rebase again

**Comment by DanielLaberge** [comment]
Alright. Just tell me when.

In the meantime, do you (or anyone else) have any clue on how to fix this
group by bug?

I wouldn't mind trying to fix it, but I'm not familiar with the codebase
enough and I'd need a couple clues...

On Wed, Mar 11, 2015 at 9:28 PM, Alexander Zaytsev <notifications@github.com

**Pull request merged**